### ISSUE TITLE
Compatibility with Scispacy

### ISSUE THREAD

**Issue by Aminolroaya** [jenojp/negspacy#33]
I am using the versions 
spaCy   3.0.3
negspacy 1.0.0
scispacy 0.4.0

I think the current version of negspacy is not compatible with scispacy. I already read the [issue ](https://github.com/jenojp/negspacy/issues/16) but I think it works with the previous negspacy version. I also tried other models of scispacy like `en_core_sci_sm` but got the same error:

```
ValueError: [E002] Can't find factory for 'negex' for language English (en). This usually happens when spaCy calls `nlp.create_pipe` with a custom component name that's not registered on the current language class. If you're using a Transformer, make sure to install 'spacy-transformers'. If you're using a custom component, make sure you've added the decorator `@Language.component` (for function components) or `@Language.factory` (for class components).

Available factories: attribute_ruler, tok2vec, merge_noun_chunks, merge_entities, merge_subtokens, token_splitter, parser, beam_parser, entity_linker, ner, beam_ner, entity_ruler, lemmatizer, tagger, morphologizer, senter, sentencizer, textcat, textcat_multilabel, en.lemmatizer
```

My code is as follows:
```
 nlp = spacy.load("en_core_sci_md")
    ts = termset("en_clinical")
    ts.add_patterns({'preceding_negations': ["nor","non"]})
    nlp.add_pipe(
        "negex",
        config={
            "neg_termset":ts.get_patterns(),
            "chunk_prefix": ["no"],
        },
        last=True,
    )

    for text in df['Cons']:
        doc = nlp(str(text))
        ..........
```
Can you help?

**Comment by jenojp** [comment]
Just double checking that you imported negspacy above with ‘’’from negspacy.negation import Negex’’’ as well as pip installed the latest scispacy language model?

**Comment by Aminolroaya** [comment]
Yes! but in my case, it looks like the code only uses "from negspacy.termsets import termset". When I added "from negspacy.negation import Negex", I got the following error 

`RuntimeError: [E896] There was an error using the static vectors. Ensure that the vectors of the vocab are properly initialized, or set 'include_static_vectors' to False.`


BTW. I transferred the code to another project and download  "en_core_web_sm" (without loading it), and the code weirdly 
 ran without error. I just wanted to inform you for the cross checking.
